src/player_hardcoded_effects.cpp

```diff
@@ -1001,8 +1001,12 @@ void player::hardcoded_effects( effect &it )
         }
 
         // TODO: Move this to update_needs when NPCs can mutate
+        bool compatible_weather_types = (g->weather == WEATHER_CLEAR || g->weather == WEATHER_SUNNY
```

Comment:
I fear gorgon can complain on this (and closing) bracket: there should bean empty space inside brackets, like `( FOO )` They ain't needed, anyway, and can be safely removed.

Reply:
ah, just being cautious.